server.js

```diff
@@ -1,16 +1,32 @@
 const pg = require('pg');
 const express = require('express');
 //const bodyParser = require('body-parser');
-//const requestProxy = require('express-request-proxy');
+const requestProxy = require('express-request-proxy');
 const PORT = process.env.PORT || 3000;
 const app = express();
-const conString = process.env.DATABASE_URL || 'postgres://postgres:DeltaV@localhost:5432/chow';
+const conString = process.env.DATABASE_URL;
+const recipeApiId =  process.env.RECIPE_API_ID;
+const recipeApiKey = process.env.RECIPE_API_KEY;
+
 const client = new pg.Client(conString);
 client.connect();
 client.on('error', err => console.error(err));
 
 app.use(express.static('./public'));
 
+function proxyEdamam(request, response){
+  request.url = request.url.replace(/%2c/gi, ',');
```

Comment:
This is the sort of hack that would be good to document either in a comment (`// HACK: <reason>` is common) or at least in the PR, acknowledging the problem you encountered and how you worked around it.